tests/test_api.py

```diff
@@ -69,6 +69,60 @@ def test_get_version(self):
         res = self.assertGetResponse("/version", requests.codes.ok, headers=get_auth_header())
         self.assertEquals(res.json['version_info']['version'], os.environ['DSS_VERSION'])
 
+    @testmode.standalone
+    def test_put_file_pattern(self):
```

Comment:
These tests belong in `test_file.py`

Reply:
I was asked to put them here, though I'm not sure if it was a strong suggestion.  I'll bring it up over slack.

Reply:
`test_file.py` works!  Migrated the tests over.